example-ideal.txt

```diff
@@ -0,0 +1,8 @@
+**Example Program in JRPicobot
+0 Wo -> W
+0 Wx So -> S
```

Comment:
It’s a little confusing that in the original Picobot, `x` means open, and here it means blocked.